ansible_collections/arista/avd/examples/campus-fabric/intended/configs/SPINE1.cfg

```diff
@@ -0,0 +1,239 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname SPINE1
+ip name-server vrf MGMT 8.8.4.4
+ip name-server vrf MGMT 8.8.8.8
+!
+ntp server vrf MGMT pool.ntp.org
+ntp server vrf MGMT time.google.com prefer
+!
+spanning-tree mode mstp
+no spanning-tree vlan-id 4093-4094
+spanning-tree mst 0 priority 4096
+!
+aaa authentication policy local allow-nopassword-remote-login
+aaa authorization exec default local
+!
+no enable password
+no aaa root
+!
+username admin privilege 15 role network-admin secret sha512 $6$eucN5ngreuExDgwS$xnD7T8jO..GBDX0DUlp.hn.W7yW94xTjSanqgaQGBzPIhDAsyAl9N4oScHvOMvf07uVBFI4mKMxwdVEUVKgY/.
+username arista privilege 15 role network-admin nopassword
+!
+vlan 10
+   name L2LEAF_INBAND_MGMT
+!
+vlan 110
+   name IDF1-Data
+!
+vlan 120
+   name IDF1-Voice
+!
+vlan 130
+   name IDF1-Guest
+!
+vlan 210
+   name IDF2-Data
+!
+vlan 220
+   name IDF2-Voice
+!
+vlan 230
+   name IDF2-Guest
+!
+vlan 310
+   name IDF3-Data
+!
+vlan 320
+   name IDF3-Voice
+!
+vlan 330
+   name IDF3-Guest
+!
+vlan 4093
+   name LEAF_PEER_L3
+   trunk group LEAF_PEER_L3
+!
+vlan 4094
+   name MLAG_PEER
+   trunk group MLAG
+!
+vrf instance MGMT
+!
+interface Port-Channel1
+   description IDF1_Po51
+   no shutdown
+   switchport
+   switchport trunk allowed vlan 10,110,120,130
+   switchport mode trunk
+   mlag 1
+!
+interface Port-Channel491
+   description LEAF2A_Po11
+   no shutdown
+   switchport
+   switchport trunk allowed vlan 10,210,220,230
+   switchport mode trunk
+   mlag 491
+!
+interface Port-Channel501
+   description IDF3_AGG_Po971
+   no shutdown
+   switchport
+   switchport trunk allowed vlan 10,310,320,330
+   switchport mode trunk
+   mlag 501
+!
+interface Port-Channel551
+   description MLAG_PEER_SPINE2_Po551
+   no shutdown
+   switchport
+   switchport trunk allowed vlan 2-4094
+   switchport mode trunk
+   switchport trunk group LEAF_PEER_L3
+   switchport trunk group MLAG
+!
+interface Ethernet1
+   description LEAF1A_Ethernet51
+   no shutdown
+   channel-group 1 mode active
+!
+interface Ethernet49/1
+   description LEAF2A_Ethernet1/1
+   no shutdown
+   channel-group 491 mode active
+!
+interface Ethernet50/1
+   description LEAF3A_Ethernet97/1
+   no shutdown
+   channel-group 501 mode active
+!
+interface Ethernet51/1
+   description LEAF3B_Ethernet97/1
+   no shutdown
+   channel-group 501 mode active
+!
+interface Ethernet55/1
+   description MLAG_PEER_SPINE2_Ethernet55/1
+   no shutdown
+   channel-group 551 mode active
+!
+interface Ethernet56/1
+   description MLAG_PEER_SPINE2_Ethernet56/1
+   no shutdown
+   channel-group 551 mode active
+!
+interface Loopback0
+   no shutdown
+   ip address 1.1.1.1/32
```

Comment:
this is a public routable IP, which we should not be using in examples.

Reply:
changed to 172.16.1.0/24